assets/js/buddyforms.js

```diff
@@ -248,6 +248,9 @@ function BuddyForms() {
     function addValidationMinLength() {
         jQuery.validator.addMethod("minlength", function (value, element, param) {
             var count = value.length;
+            if( value ===""){
```

Comment:
@marin250189 what happens if the field using this validation is required and is empty?